src/components/core/core.js

```diff
@@ -203,6 +204,17 @@ export default class Core extends UIObject {
     this.mediaControl.show()
   }
 
+  handleWindowResize(event) {
+    let orientation = ($(window).width() > $(window).height()) ? 'landscape' : 'portrait'
+    if (this._screenOrientation === orientation) return
+    this._screenOrientation = orientation
+
+    this.trigger(Events.CORE_SCREEN_ORIENTATION_CHANGED, {
+      event: event,
+      orientation: this._screenOrientation
```

Comment:
@leandromoreira it is already an object with two properties : `event` which is the window resive event and `orientation` which is a string which equals 'landscape' or 'portrait'.